DANS-KNAW/dd-verify-migration
# DD-728: Retrieve actual files from Data Station

Fixes DD-728: Retrieve actual files from Data Station

# Description of changes

* the required default bundle name `hibernate` moved from easy_db to `verificationDatabase` as that one is used for all sub-commands.
* implemented the new sub-command.

# How to test

Though each sub-command uses either deasy or dataverse, both must be up and running due to the single configuration. 

* in the root of easy-dtap: `start-preprovisioned-box.py`
* in the root of dd-verify-file-migration: 
  * once: run `start-env.sh` and change `etc/config.yml`:
    * dataverse baseUrl: 'https://demo.archaeology.datastations.nl/'
    * the apiKey for dataverse and password for both databases
  * per test, with VPN to reach the demo server (each command line in another terminal, or the second command in Intellij by running DdVerifyFileMigrationApplication configured with the specified arguments):

        rm -rf data;start-hsqldb.sh
        start.sh load-from-dataverse [-d doi:10.17026/dans-zta-6jwm] etc/config.yml

* [x] Or build, deploy on deasy, adjust dataverse url/apiKey configuration as above and:

      dd-verify-file-migration load-from-dataverse  [-d doi:10.17026/dans-zta-6jwm] /etc/opt/dans.knaw.nl/dd-verify-file-migration/config.yml > verify.log

  You may want to switch off info logging for `org.hibernate.engine.internal.StatisticalLoggingSessionEventListener`

# Related PRs

* [x] [dd-dans-dataverse-client-lib](https://github.com/DANS-KNAW/dans-dataverse-client-lib/pull/14) needs to support  `DataFile.original*` for [10.17026/dans-xtz-qa6j](https://demo.archaeology.datastations.nl/api/datasets/export?exporter=dataverse_json&persistentId=doi%3A10.17026/dans-xtz-qa6j) merged, but no new version yet.

# Notify
@DANS-KNAW/dataversedans
